websites/D/Discord BotList/presence.ts

```diff
@@ -1,121 +1,150 @@
 const botPresence = new Presence({
-    clientId: "819122551435296818"
-  }),
+  clientId: "819122551435296818"
+}),
   botBrowsing = Math.floor(Date.now() / 1000);
 
 botPresence.on("UpdateData", async () => {
   const botData: PresenceData = {
-      largeImageKey: "logo",
-      startTimestamp: botBrowsing
-    },
+    largeImageKey: "logo",
+    startTimestamp: botBrowsing
+  },
     botPage = document.location.pathname,
     botHost = document.location.hostname,
     buttons = await botPresence.getSetting("buttons"),
     sSubmit = await botPresence.getSetting("submitP");
 
   if (botHost == "discord-botlist.eu") {
-    if (botPage == "/") {
-      botData.details = "Browsing";
-    } else if (botPage == "/search") {
+    if (botPage === "/") 
+
+      botData.details = "Browsing home";
+
+     else if (botPage === "/search") {
+
```

Comment:
You don't need this many empty lines, you can just put the code on the same line as the if check